### PR TITLE
zh-TW and zh-CN translation update for 3.0 version

### PR DESCRIPTION
**Pull request by ky0nch3ng**
zh-TW and zh-CN translation update for 3.0 version

My ID has been changed， please renew the contributors list for zh-TW. thx

### CODE REVIEW

File: OpenIV/Resources/Languages/zh_CN/LIP.xml [hunk 1]
```diff
@@ -1,9 +1,6 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<LanguageInterfacePack version='290' ID='2052' active="true">
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<LanguageInterfacePack version='300' ID='2057' active="true">
 	<Contributors>
-		<Contributor link="http://openiv.com/">珑璃</Contributor>
+		<Contributor link="http://openiv.com/">OpenIV 团队</Contributor>
```
Comment (GooD-NTS) [inline OpenIV/Resources/Languages/zh_CN/LIP.xml]: You put "OpenIV Team" here. But, we didn't contribute for this language. Please change it.

File: OpenIV/Resources/Languages/zh_TW/LIP.xml [hunk 2]
```diff
@@ -1,9 +1,6 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<LanguageInterfacePack version='290' ID='1028' active="true">
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<LanguageInterfacePack version='300' ID='2057' active="true">
 	<Contributors>
-		<Contributor link="http://openiv.com/">ky0ncheng</Contributor>
+		<Contributor link="http://openiv.com/">OpenIV 團隊</Contributor>
```
Comment (GooD-NTS) [inline OpenIV/Resources/Languages/zh_TW/LIP.xml]: You put "OpenIV Team" here. But, we didn't contribute for this language. Please change it.

**Comment by ky0nch3ng** [comment]
@GooD-NTS 

Edit <Contributor> value is completed.

```
<LanguageInterfacePack version='290' ID='1046' active="true">	
```
I don't sure `ID='1046'` in this code, keep value is same as old version.

**Comment by GooD-NTS** [comment]
@ky0nch3ng you can check the ID here https://github.com/OpenIV-Team/OpenIV-Languages/blob/master/LANGUAGES.md (The Chinese language variations is listed at the end of the list)

Please let me know if everything is correct and I will merge pull request.

**Comment by ky0nch3ng** [comment]
OK，It's correct and you can merge this PR.

**Pull request merged**